app/controllers/registrations_controller.rb

```diff
@@ -0,0 +1,8 @@
+# https://github.com/heartcombo/devise/wiki/How-To:-Allow-users-to-edit-their-account-without-providing-a-password
+class RegistrationsController < Devise::RegistrationsController
+  protected
+
+  def update_resource(resource, params)
+    resource.update_without_password(params)
+  end
+end
```

Comment:
After updating the resource, the default redirect is to the root page. In my opinion, the profile page of the user would make more sense! This can be achieved via:
```ruby
def after_update_path_for(resource)
    profile_path(resource)
  end
```